templates/ts-standby/.prettierrc

```diff
@@ -0,0 +1 @@
+{ "printWidth": 100 }
```

Comment:
Hey, why is 100 here, while 120 in `.editorconfig`? Thanks

Also I was told prettier will be only able to either work with `.prettierrc` or `.editorconfig`, not with both